workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -68,19 +68,14 @@ public void Awake()
                     ? WorkerRegistry.CreateWorker<UnityClient>($"{workerType}-{Guid.NewGuid()}", new Vector3(0, 0, 0))
                     : WorkerRegistry.CreateWorker(workerType, workerId, new Vector3(0, 0, 0));
 
-                if (worker == null)
-                {
-                    return;
-                }
-
                 Workers.Add(worker);
 
                 connectionConfig = ConnectionUtility.CreateConnectionConfigFromCommandLine(commandLineArgs);
             }
 
             if (World.AllWorlds.Count <= 0)
             {
-                Debug.LogError(
+                throw new ArgumentException(
```

Comment:
Please make new exception types for specific errors rather than use the base ones.
Also this isn't an argument exception.